### PR TITLE
feat: Update formatting of code, align error handling

### PR DESCRIPTION
**Pull request by pim-simons**
Formatted all code so everything is aligned, aligned error handling as well.

### CODE REVIEW

File: src/Arcus.Scripting.DevOps/Scripts/Set-AzDevOpsArmOutputs.ps1 [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+    [switch][parameter(Mandatory = $false)] $UpdateVariableGroup = $false,
+    [switch][parameter(Mandatory = $false)] $UpdateVariablesForCurrentJob = $false
 )
 
-function Add-VariableGroupVariable()
-{
+function Add-VariableGroupVariable() {
     [CmdletBinding()]
     param
     (
```
Comment (stijnmoreels) [inline src/Arcus.Scripting.DevOps/Scripts/Set-AzDevOpsArmOutputs.ps1:11]: Should also be on the same line, then? Like `param(`

File: src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1 [hunk 2]
```diff
@@ -76,10 +76,10 @@ Export-ModuleMember -Function Set-AzKeyVaultSecretFromFile
 
 function Set-AzKeyVaultSecretAsBase64FromFile {
     param (
```
Comment (stijnmoreels) [inline src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1:78]: Most of the time without space, so maybe change these in the file also without space? `param(`?

**Review by stijnmoreels — APPROVED** [review]
Well, those were some well-needed code formattings! 👀 😄 I can definitely agree with them for scripting. Amazing job, thx for this! 🎉

File: src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1 (hunk 2 above)
Comment (pim-simons) [reply src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1]: Good one, fixed in [`901d049` (#424)](https://github.com/arcus-azure/arcus.scripting/pull/424/commits/901d0490716209e4a3a8a143f1037673457ca6bd)

File: src/Arcus.Scripting.DevOps/Scripts/Set-AzDevOpsArmOutputs.ps1 (hunk 1 above)
Comment (pim-simons) [reply src/Arcus.Scripting.DevOps/Scripts/Set-AzDevOpsArmOutputs.ps1]: Good one, fixed in [`901d049` (#424)](https://github.com/arcus-azure/arcus.scripting/pull/424/commits/901d0490716209e4a3a8a143f1037673457ca6bd)

**Pull request merged**